client/my-sites/stats/activity-log/index.jsx

```diff
@@ -373,7 +383,12 @@ export default connect(
 			rewindStatusError: getRewindStatusError( state, siteId ),
 
 			// FIXME: Testing only
-			isPressable: get( state.activityLog.rewindStatus, [ siteId, 'isPressable' ], false ),
+			isPressable: true,
+			logs: get( state, [
```

Comment:
If we have a `logItems` selector, it should be used. If not, we can follow-up in another PR.